var/spack/repos/builtin/packages/mofem-cephas/package.py

```diff
@@ -0,0 +1,128 @@
+##############################################################################
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+
+class MofemCephas(CMakePackage):
+    """mofem-cephas core library"""
+
+    @property
+    def root_cmakelists_dir(self):
+        """The relative path to the directory containing CMakeLists.txt
+
+        This path is relative to the root of the extracted tarball,
+        not to the ``build_directory``. Defaults to the current directory.
+
+        :return: directory containing CMakeLists.txt
+        """
+        return 'mofem'
+
+    homepage = "http://mofem.eng.gla.ac.uk"
+    url = "https://likask@bitbucket.org/likask/mofem-cephas.git"
+
+    maintainers = ['likask']
+
+    version('0.8.3', git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.3', submodules=True)
+    version('develop',
+        git='https://likask@bitbucket.org/likask/mofem-cephas.git',
+        branch='develop')
+
+    # Obsolete:
+    # Variants of packages installed as extensions with MoFEM, to indicate this
+    # specific type of variant prefox with_ is added.
+    # variant('with_adol-c', default=False,
+    #         description='Install ADOL-C with MoFEM')
+    # variant('with_tetgen', default=False,
+    #         description='Install TetGen with MoFEM')
+    # variant('with_med', default=False,
+    #         description='Install MED with MoFEM')
+
+    # This option can be only used for development of core lib
+    variant('copy_user_modules', default=True,
+        description='Copy user modules directory instead linking to source')
+    variant('adol-c', default=True, description='Compile with Adol-C')
+    variant('tetgen', default=True, description='Compile with Tetgen')
+    variant('med', default=True, description='Compile with Med')
+    variant('slepc', default=True, description='Compile with Slepc')
+    variant('documentation', default=False, description='Install doxygen')
```

Comment:
sorry, i realised that my previous comment was ambiguous. What I wanted to suggest is to change description to `Compile with documentation` and perhaps call the variant `doc` just to make shorter.